app/javascript/crayons/Buttons/__stories__/Buttons.mdx

```diff
@@ -14,35 +14,50 @@ That is: `<CTA />` or `<Link />` should be used where navigation is elicited, an
+When combined with the _Primary_ or the _Secondary_, the _Default_ variant should be used for any remaining actions.
+
+Whenever used, make sure it has sufficient color contrast against the background it's used on.
+
 ### Primary
 
-Primary button has special look: high contrast (against rest of the view/component) and it's filled with accent color. 
+The _Primary_ variant has special look: high contrast (against rest of the view/component) and it's filled with accent color. 
 
-Ideally there should be only one Primary button per entire component, or sometimes even per entire view.
-Primary buttons should be used for the principal call to action on the page.
+Ideally there should be only one _Primary_ button per entire component, or sometimes even per entire view.
+Primary buttons should be used for the principal action on the page.
 
-### Default
+### Secondary
+
+The _Secondary_ variant is visually in between _Primary_ and _Default_ buttons. It's not as loud as _Primary_ button but definitely stands out more than _Default_ button.
+It's especially useful when you want your button to be more distinguishable from its surroundings without making it super loud (like _Primary_). 
+A good example would be a component with lots of text. The _Default_ button could be potentially missed there (as it uses similar colors as regular texts).
+
+Whenever you have doubts between using the _Secondary_ vs the _Default_ variants, the rule of thumb should be:
+
+> Use the _Default_ variant by default :D unless it's not distinguishable enough from its surroundings.
 
-Default variant is less prominent, and therefore can be used for less prominent actions.
-Default buttons are visually light (greyish) and you can have as many of them as you want (within common sense :D). 
-They can (but don't have to) be used in conjunction with a Primary button. In that case, default variant should be used for any remaining actions.
+When combined with the _Primary_ button, the _Secondary_ variant should be used for any remaining or alternative actions.
 
 ## Configuration
 
 ### Destructive actions
 
 For actions that could have destructive effects on the user’s data (for example, *delete* or *remove*) we have a special style.
-This style can be applied by using `destructive` prop and it will work for both: primary and default variants. Take care when using the default variant to ensure that the red text has sufficient color contrast against the background where you are using it.
+This style can be applied by using `destructive` prop and it will work with two variants: _Primary_ and _Default_ (as it doesn't really make sense to have _Secondary_ variant for destructive buttons).
 
 ### Icons
 
-Icon can be placed in front of a label to both clarify an action and call attention to a button. 
+Icon can be placed in front of a label to both clarify an action but also to make button stand out more.
```

Comment:
Quick suggestion: let's make "icon" plural here and add the word "the"!
```suggestion
Icons can be placed in front of a label to both clarify an action but also to make the button stand out more.
```

Reply:
But you can only have one icon in a button?